cvat/apps/dataset_manager/formats/registry.py

```diff
@@ -45,29 +45,30 @@ def __call__(self, *args, **kwargs):
     target.DISPLAY_NAME = (display_name or klass.DISPLAY_NAME).format(
         NAME=name, VERSION=version, EXT=ext)
     assert all([target.NAME, target.VERSION, target.EXT, target.DISPLAY_NAME])
+    target.DIMENSION = dimension
     target.ENABLED = enabled
 
     return target
 
 EXPORT_FORMATS = {}
-def exporter(name, version, ext, display_name=None, enabled=True):
+def exporter(name, version, ext, display_name=None, enabled=True, dimension="2d"):
     assert name not in EXPORT_FORMATS, "Export format '%s' already registered" % name
     def wrap_with_params(f_or_cls):
         t = _wrap_format(f_or_cls, Exporter,
             name=name, ext=ext, version=version, display_name=display_name,
-            enabled=enabled)
+            enabled=enabled, dimension=dimension)
         key = t.DISPLAY_NAME
         assert key not in EXPORT_FORMATS, "Export format '%s' already registered" % name
         EXPORT_FORMATS[key] = t
         return t
     return wrap_with_params
 
 IMPORT_FORMATS = {}
-def importer(name, version, ext, display_name=None, enabled=True):
+def importer(name, version, ext, display_name=None, enabled=True, dimension="2d"):
```

Comment:
I believe `2D` and `3D` should be enum values. Look at similar constants and how they are defined in server's code.

Reply:
Made changes accordingly.